packages/frontend/src/main/dialogs/ShareStreamDialog.vue

```diff
@@ -103,12 +103,13 @@
         </v-toolbar-title>
         <v-spacer></v-spacer>
         <v-switch
-          v-model="stream.isPublic"
```

Comment:
Mutating props is a big no no in Vue, it can lead to hard to debug bugs. The correct approach is to emit changes out with an event or in this case since the data comes from Apollo cache - to update the apollo cache directly